config/app/init/default.json

```
@@ -1,16 +1,18 @@
 {
   "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^1.0.0/components/context.jsonld",
   "import": [
-    "files-scs:config/app/init/base/init.json",
-    "files-scs:config/app/init/initializers/root.json"
+    "files-scs:config/app/init/base/init.json"
   ],
   "@graph": [
     {
       "comment": "These handlers are called whenever the server is started, and can be used to ensure that all necessary resources for booting are available.",
       "@id": "urn:solid-server:default:ParallelInitializer",
       "@type": "ParallelHandler",
       "handlers": [
-        { "@id": "urn:solid-server:default:RootInitializer" }
+        {
+          "comment": "This handler is here because empty arrays sometimes give unexpected behaviour.",
```

err… 🙂 is there an issue for that?

```suggestion
          "comment": "Empty placeholder so that you can add your own handlers here.",
```

---

The problem is that it's not easy to make a small reproducible example. I should probably test this again since there are locations where we can use empty arrays, but last time I tested removing the RootInitializer from the array it gave me errors. So will test again, but sometimes we run into weird issues due to how our config modify the same object from different places.